### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by joaopedroalbq**
All the features related to the project are in this branch.

Ps.: We should have chosen a more descriptive name.

### CODE REVIEW

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Great work team :clap: 
Nice job building your very first project :smile:
You folks get almost everything right, we just need to do a few changes to make your project even better. 

- [x] It's very important to have a descriptive readme file. This is the first thing that people look in your repository. Don't forget that in the future, your repositories are going to be part of your portfolio. If you need a direction on this you can make something like this one https://github.com/bolah2009/newsweek-clone/blob/master/README.md. The images are optional, but it's best practice have the name and git account of all contributors.

- [x]  There are some errors in your HTML code. Check them using this HTML [validator](https://validator.w3.org/) . Ignore the warnings.

- [x]  There is also one error in your CSS code. Check it using this CSS [validator](https://jigsaw.w3.org/css-validator/) . 

Please, kindly make these changes and submit for code review again. Happy coding :muscle: 

PS: I left some note in your code. They are all optional, but they will improve a lot the quality of your code.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Youtube Clone</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel=“stylesheet”>
+</head>
+<body>
+    <div class="navbar">
```
Comment (Torres-ssf) [inline index.html]: [OPTIONAL]
It's best practice to use the `nav` tag for the navbar. This makes your code more semantic and improves the overall quality of your code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        </div>
+
+        <div class="navbar-items">
+            <a><i class='material-icons'>apps</i></a>
+            <a><i class='material-icons'>message</i></a>
+            <a><i class='material-icons'>video_call</i></a>
+            <div class="navbar-avatar"></div>
+        </div>
+    </div>
+    <div class="main">
```
Comment (Torres-ssf) [inline index.html]: [OPTIONAL]
It's best practice to use the `main` tag for the main content of your page. The main content of your page it's the most important part of it. In this project's case, the video section

File: index.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                            <p class="video-description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Iusto omnis doloremque itaque, illo voluptas quo vitae blanditiis nesciunt cumque libero?</p>
+                        </div>
+                    </div>
+                    <div>
+                        <button class="subscribe-button" type="button">SUBSCRIBE</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="section-sidebar">
```
Comment (Torres-ssf) [inline index.html]: [OPTIONAL]
It's also best practice to use the `aside` tag for the side section. All these 3 changes are related to best practices and semantic HTML(very important topic)

**Review by Ispirett — APPROVED** [review]
Hey @jstiven01 @joaopab1337  Great! job on your first project
image. 

looks good to me you can merge and complete this project.
##### good luck on your next project.

**Pull request merged**